modules/core/src/main/scala/schema.scala

```diff
@@ -630,7 +630,7 @@ case class NullableType(
  *
  * @see https://facebook.github.io/graphql/draft/#sec-The-__Field-Type
  */
-case class Field private(
+case class Field(
```

Comment:
We're calling this from a place where Scala 3 says we can't. I don't think it needs to be `private` anyway.